gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -109,6 +109,10 @@ local function AddBindingCategory(category, parent)
 	form:Dock(TOP)
 end
 
+function HELPSCRN.IsOpen()
+	return helpframe and IsValid(helpframe) or IsValid(LocalPlayer().settingsFrame)
```

Comment:
First check is useless, because IsValid(nil) will fail too. Just to simplify it, I like to use just IsValid :) What do you think?

Reply:
I second this. See the [IsValid function](https://github.com/Facepunch/garrysmod/blob/cf262a1342ccb79831046b98452317ba1a1f9e3b/garrysmod/lua/includes/util.lua#L203)

Reply:
It's an artifact because I copied it from a few lines below. I fixed both instances.